fedmsg_meta_fedora_infrastructure/fasshim.py

```diff
@@ -1,13 +1,14 @@
 import collections
+import os
 import threading
 import socket
 import string
+import logging
```

Comment:
should we order them alphabetically? At least for the two we're adding/moving?

Reply:
👍